### PR TITLE
Enable permissionless asset creation

### PR DESCRIPTION
**Pull request by gavofyork**

### CODE REVIEW

File: polkadot-parachains/statemine-runtime/src/lib.rs [hunk 1]
```diff
@@ -148,12 +148,7 @@ parameter_types! {
 // Don't allow permission-less asset creation.
```
Comment (JoshOrndorff) [inline polkadot-parachains/statemine-runtime/src/lib.rs:148]:
```suggestion
```

**Review by apopiak — APPROVED** [review]
LGTM

**Comment by xlc** [comment]
Does those needs an update given new parachains will be live soon?

https://github.com/paritytech/cumulus/blob/a84f7866f5ce36eb246407fce1e5f813e5eff575/polkadot-parachains/statemine-runtime/src/lib.rs#L591-L594

**Review by joepetrowski — APPROVED** [review]
LGTM for release on Kusama providing cross-chain asset transfers are not enabled in the same upgrade in order to keep any Assets/Uniques issues local to Statemine.

**Review by apopiak — APPROVED** [review]
LGTM

**Review by shawntabrizi — APPROVED** [review]

**Pull request merged**